lib/ransack.rb

```diff
@@ -1,3 +1,4 @@
+require 'active_support/dependencies/autoload'
```

Comment:
Just requiring this explicitly since `on_load` does get used further down in this file (and I suppose should be considered in any other files that get required from here).